.circleci/config.yml

```diff
@@ -1,47 +1,26 @@
 version: 2.1
 
-parameters:
-  release-name:
-    type: string
-    default: "runner-init"
```

Comment:
I think we decided not to use an different repo for dev builds? The codebase is public, so no point for this parameter really. We can add this back if needed in the future.

Reply:
Also, we might want to publish dev tags in the future too. If we want to incorporate them in smoke tests and such.